src/common/fontcmn.cpp

```diff
@@ -711,7 +781,7 @@ wxString wxNativeFontInfo::ToString() const
 
 void wxNativeFontInfo::Init()
 {
-    pointSize = 0;
+    pointSize = 0.0;
```

Comment:
How about `0.0f` or `0.f` rather than `0.0`? At least to be consistent with other suffixed values.

Reply:
Yes, `f` was supposed to be there. Not sure how did I drop it, restored now, thanks.